addons/packages/velero/1.9.2/package.yaml

```diff
@@ -0,0 +1,548 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: velero.community.tanzu.vmware.com.1.9.2
+spec:
+  refName: velero.community.tanzu.vmware.com
+  version: 1.9.2
+  releaseNotes: "velero 1.9.2 https://github.com/vmware-tanzu/velero/releases/tag/v1.9.2"
+  licenses:
+    - "Apache 2.0"
+  template:
+    spec:
+      fetch:
+        - imgpkgBundle:
+            image: projects-stg.registry.vmware.com/tce/velero@sha256:a3256437d91b3fa63177832d346753c1876cc40c30c7ea4ff0ac876aed4839fb
```

Comment:
Be sure to push the package to `projects.registry.vmware.com` once you're satisfied that everything is working.

Reply:
Got it.
Modified.